src/magazine.py

```diff
@@ -40,39 +47,61 @@ def download_magazine(self, id):
             done = False
             while done is False:
                 status, done = downloader.next_chunk()
-
+                
         except HttpError as error:
             print(f"An error occurred: {error}")
             file = None
+            
         if file is not None:
-            payload = json.dumps(
-                {
-                    "bucket": UPLOAD_BUCKET,
-                    "image": "data:application/pdf;base64,"
-                    + base64.b64encode(file.getvalue()).decode("utf-8"),
-                }
-            )
-            return requests.post(
-                "https://upload.cornellappdev.com/upload/",
-                payload,
-            ).text
+            pdf = pdfium.PdfDocument(file)
+            first_page = pdf[0]
+            pil_image = first_page.render().to_pil().resize((150,220))
+
+            imageBytes = io.BytesIO() #create bytes stream to hold image
+            pil_image.save(imageBytes, format='png')
+            
+            image_payload = json.dumps(
+            {
+            "bucket": "volume",
+            "image": "data:image/png;base64,"
+            + base64.b64encode(imageBytes.getvalue()).decode("utf-8")})
+            
+            image_response = requests.post("https://upload.cornellappdev.com/upload/",image_payload).text
+
+            pdf_payload = json.dumps(
+                {"bucket": UPLOAD_BUCKET,
+                "image": "data:application/pdf;base64,"
+                + base64.b64encode(file.getvalue()).decode("utf-8")})
```

Comment:
let's make the way we present dictionaries consistent. i think the best way is

```
json.dumps({
    "bucket": "volume",
    "image": "data:image/png;base64,"
    + base64.b64encode(imageBytes.getvalue()).decode("utf-8")
})
```
or
```
json.dumps(
    {
        "bucket": "volume",
        "image": "data:image/png;base64,"
        + base64.b64encode(imageBytes.getvalue()).decode("utf-8")
    }
)
```